### DISCUSSION TITLE
Linting

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#356] [General]
Do you use `yarn lint:fix`?

Every time there is a new code change I face a lot of merge conflict issues due to code formatting which I think could be avoided by simply using the already available `lint:fix` script.

Ideally would be great to have a more robust code formatter (prettier or something else) but we discussed this already.

A good code consistent is crucial.

Thoughts.

**Comment by Vadorequest** [comment]
I don't use it automatically, sorry about the inconvenience.

I don't experience much "code formatting" issues/conflicts though, and we have 3 projects based on both NRN's presets. 🤔

**Reply by samuelcastro** [reply]
Because you have your own code styling and IDE configuration which is different from others I guess.
